functional/sev/run.sh

```diff
@@ -0,0 +1,337 @@
+#!/bin/bash
+# Copyright 2022 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -eE
+set -o nounset
+set -o pipefail
+
+#trap cleanup EXIT
+
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m'
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+tests_repo_dir="$(realpath "$script_dir/../..")"
+
+export EDK2_REPO_DIR="${HOME}/edk2"
+export DOCKER_USER="rksavino"
+
+export KBS_DB_HOST="127.0.0.1"
+export KBS_DB_USER="root"
+export KBS_DB_PW="root"
+export KBS_DB="simple_kbs"
+export KBS_DB_TYPE="mysql"
+
+esudo() {
+  sudo -E PATH=$PATH "$@"
+}
+
+modify_kata_config() {
+  esudo mkdir -p /etc/kata-containers/
+  esudo install -o root -g root -m 0640 /usr/share/defaults/kata-containers/configuration.toml /etc/kata-containers
+
+  # Enable full debug
+  esudo sed -i -e 's/^# *\(enable_debug\).*=.*$/\1 = true/g' /etc/kata-containers/configuration.toml
+  
+  # kernel, initrd, image, firmware
+  esudo sed -i -e 's/kernel = "\(.*\)"/kernel = "\/usr\/share\/kata-containers\/vmlinuz-sev.container"/g' /etc/kata-containers/configuration.toml
+  esudo sed -i -e 's/initrd = "\(.*\)"/initrd = "\/usr\/share\/kata-containers\/kata-containers-initrd.img"/g' /etc/kata-containers/configuration.toml
+  esudo sed -i -e 's/image = "\(.*\)"/#image = ""\ninitrd = "\/usr\/share\/kata-containers\/kata-containers-initrd.img"/g' /etc/kata-containers/configuration.toml
+  esudo sed -i -e "s|^\(firmware = \"\).*\"|\1${EDK2_REPO_DIR}/Build/AmdSev/DEBUG_GCC5/FV/OVMF.fd\"|g" /etc/kata-containers/configuration.toml
+  
+  # kernel_params
+  esudo sed -i -e 's/kernel_params = "\(.*\)"/kernel_params = " agent.log=debug initcall_debug agent.config_file=\/etc\/default\/agent.cfg console=ttyAMA0,115200 console=tty highres=off console=ttyS0"/g' /etc/kata-containers/configuration.toml
```

Comment:
I'm not sure we need all the extra console stuff anymore.

Reply:
This will get removed with 5013. I'll also remove them from the kernel params in the `sev-snp-measure` call when 5013 is in.